bundles/org.openhab.binding.modbus/README.md

```diff
@@ -122,6 +122,7 @@ Advanced parameters
 | `timeBetweenTransactionsMillis` |          | integer | `60`               | How long to delay we must have at minimum between two consecutive MODBUS transactions. In milliseconds.                                                            |
 | `timeBetweenReconnectMillis`    |          | integer | `0`                | How long to wait to before trying to establish a new connection after the previous one has been disconnected. In milliseconds.                                     |
 | `connectMaxTries`               |          | integer | `1`                | How many times we try to establish the connection. Should be at least 1.                                                                                           |
+| `afterConnectionDelayMillis`    |          | integer | `0`                | Delay between connection and first modbus operation, reflected in milliseconds.                                                                                    |
```

Comment:
Suggesting to have same description as in UI, similar to the other parameters.

```suggestion
| `afterConnectionDelayMillis`    |          | integer | `0`                | Connection warm-up time. Additional time which is spent on preparing connection which should be spent waiting while end device is getting ready to answer first modbus call. In milliseconds.                                                            |
```

Reply:
Resolved in https://github.com/openhab/openhab-addons/pull/12642/commits/9124e01e0c8601fb58d1cd35b862b8954f106c48